applications/TrilinosApplication/tests/test_mpi_communicator.py

```diff
@@ -62,7 +62,7 @@ def _read_model_part_mpi(self,main_model_part):
         model_part_io.ReadModelPart(main_model_part)
 
         ## Construct and execute the Parallel fill communicator
-        ParallelFillCommunicator = KratosTrilinos.ParallelFillCommunicator(main_model_part.GetRootModelPart())
+        ParallelFillCommunicator = KratosMPI.ParallelFillCommunicator(main_model_part.GetRootModelPart())

```

Comment:
with this change I think the import of the TrilinosApp is no longer needed in this file
=> this will make it easier when moving it to the core

Reply:
Done, but I think most tests in this file are for the outdated python interface, so I'm not sure we'll move it anyway.